ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.less

```diff
@@ -20,14 +21,20 @@
     }
 
     .search-form-label {
-        .is-bold();
+        font-weight: bold;
     }
+
+    .choice-actions {
+        position: absolute;
+        right: 10px;
```

Comment:
@andrewkfiedler should this be a multiple of `@minimumSpacing`?

Reply:
Updated to use @minimumSpacing